### PR TITLE
Adding a plot of received, sent friend requests each month

### LINKED ISSUE #7: Friends > Did you send them a request or they did?
This is an addition over the current feature.
 How many friends you make, based on received friend requests, and requests you sent. For each day, you have 2 entries, friends you made who sent you requests, and friends you sent a request to. (ed

### PR DESCRIPTION
**Pull request by parasjain-12**
Fixes #7 
I am sorry for creating a new pull request. In this PR i have updated README, uploaded the plot and the code.

### CODE REVIEW

**Comment by parasjain-12** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -142,6 +142,28 @@ Plot all messages so far, with new messages each hour.
...
+
+<details>
+  <summary>Plot all the friend request received,send and total friend so far.</summary>
+  </details>
+  
+Plot the cumalative friend you may every month,friend request send and friend request received.
+![](images/MonthwiseFriend.png)
+
+<details>
+  <summary>Plot the no.of friend request received,friend request send and total friend each month</summary>
```
Comment (techytushar) [inline README.md]: change `no. of` to `number of`

**Comment by techytushar** [comment]
Just a small change otherwise LGTM, nice work.

File: README.md (hunk 1 above)
Comment (kaustubhhiware) [reply README.md]: `cumulative`, instead of `cumalative`

File: README.md [hunk 2]
```diff
@@ -170,11 +192,17 @@ Plot count of different reactions to posts
 
 ## Observations
 
-1. There is a spike in friends made in March (Election season) and July (new juniors, much higher spike).
 
-2. I tend to message less during exams (Feb, Apr, Sep, Nov).
 
-3. Highest number of messages sent at 9 and 11 pm, confirming with calls from home come at 10pm. Almost no messages shared between 3am-7am.
+1. In the first month I haven't received any friend request.
```
Comment (kaustubhhiware) [inline README.md]: Please do not add your observations to the top, add them at the end.

Comment (parasjain-12) [reply README.md]: Done

**Comment by parasjain-12** [comment]
@kaustubhhiware  I took the reference from your code only (no.of friend monthwise) in your code also it was 13. Even if I increase the index of friend request received per month  I am not getting any value, same for send request.

**Comment by parasjain-12** [comment]
@roopalJazz  done

**Comment by kaustubhhiware** [comment]
@parasjain-12 you're still giving 13 as the range. That's where you're going wrong. You need to give the number of months active on fb as the list length instead of 13, in percentage_received_friend. 
Still needs work.

**Comment by parasjain-12** [comment]
I have a doubt in your code of plot_friend.py you also used 13. Can you please tell me why. And even when I increase the index of monthwise in plot_friend.py the values I am getting is zero . I am not able to figure out what is happening here

**Comment by kaustubhhiware** [comment]
Yeah, because it's supposed to be month wise in plot_friends.py. The plot you're comparing to is different. 
There is no plot similar to the one I'm requesting in the repository yet.

**Comment by kaustubhhiware** [comment]
Good work with the new plot. Just try to play around and make it visually better.
If needed replace bar plot with line plot or try changing bar width. 

Once you do that, I'll merge it soon.

**Comment by kaustubhhiware** [comment]
@techytushar could you check if the error you were getting is resolved?

@parasjain-12 if possible, do you see some sort of patterns in `Compare.png`? Can you share them here?
Last thing from my side, I don't mind merging this in if the error in script execution is resolved.

**Comment by techytushar** [comment]
Only one of the plots is being displayed.

screenshot from 2018-07-09 13-22-55

**Comment by techytushar** [comment]
Nie work on resolving the errors, just a minor query. My first plot looks like this: 
screenshot from 2018-07-09 15-06-09
But the second plot looks like this, is it supposed to look like this?
screenshot from 2018-07-09 15-05-16

**Comment by parasjain-12** [comment]
@techytushar for my data the second plot looks. Can you verify your data which you have received from Facebook. Does it contain the friend request received data.

**Comment by techytushar** [comment]
@parasjain-12 yes, it contains some data, around 41 requests but those are all in different months, so that might be the reason for this type of plot. Looks good to me, nice work :+1:

**Comment by parasjain-12** [comment]
@techytushar I verified the data for my friends It all look good

**Comment by kaustubhhiware** [comment]
Cool.

Just squash this PR into 2-3 commits, and I'll merge it. 22 commits is a bit too much. Refer [stackoverflow](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

**Comment by parasjain-12** [comment]
@roopalJazz  I have one question Why I am not able to see my code and graph in the master branch.

**Comment by roopalJazz** [comment]
@parasjain-12 I have done the changes.

**Pull request merged**